server/utils/utils.ts

```diff
@@ -21,3 +21,9 @@ export const initialiseName = (fullName?: string): string | null => {
   const array = fullName.split(' ')
   return `${array[0][0]}. ${array.reverse()[0]}`
 }
+
+export function asArray(value?: string | string[]): string[] {
```

Comment:
Not withstanding my comment about "utils" (that is more about naming conventions and organising code etc), this is a neat function 👍 

This is really minor, and we don't need it, but I wonder if we could make this function even better/ more useful in the future by using generics.
Would this method signature work do you think?
```
export function asArray<T>(value?: T | T[]): T[] {
```

Reply:
Nice suggestion. I've updated.